forkable/forkdb.go

```diff
@@ -85,10 +85,8 @@ func (f *ForkDB) SetLogger(logger *zap.Logger) {
 // unknown behaviour if it was already set ... maybe it explodes
 func (f *ForkDB) TrySetLIB(headRef bstream.BlockRef, previousRefID string, libNum uint64) {
 	if headRef.Num() == bstream.GetProtocolFirstStreamableBlock {
-		f.libRef = bstream.NewBlockRef(previousRefID, bstream.GetProtocolGenesisBlock)
-		libNum = bstream.GetProtocolGenesisBlock
-
-		f.logger.Debug("candidate LIB received is first streamable block of chain, assuming it's the new LIB", zap.Stringer("lib", f.libRef))
```

Comment:
I feel we should keep the log line here, it's an edge case and it make sense to have some report about it.